mobile/native/src/ln_dlc/mod.rs

```diff
@@ -525,6 +532,136 @@ pub fn get_usable_channel_details() -> Result<Vec<ChannelDetails>> {
+                    position::handler::update_position_after_dlc_closure(Some(filled_order))?;
+                } else if matches!(
+                    dlc_channel.state,
+                    SubChannelState::CloseOffered(_) | SubChannelState::CloseAccepted(_)
+                ) {
+                    recover_dlc().await?;
+                } else {
+                    tracing::warn!(
+                        "The DLC is in a state that can not be recovered. Removing position."
+                    );
+                    // maybe a left over after a force-closure
+                    position::handler::update_position_after_dlc_closure(None)?;
+                }
+            }
+        }
+        (None, Some(dlc_channel)) => {
+            if matches!(dlc_channel.state, SubChannelState::OffChainClosed) {
+                tracing::info!("DLC channel and position are in sync");
+            } else {
+                tracing::warn!("Found unexpected sub channel state {:?}", dlc_channel.state);
+                if matches!(dlc_channel.state, SubChannelState::Signed(_)) {
+                    tracing::warn!("Trying to recover position from order");
+                    match order::handler::order_filled() {
+                        Ok(order) => {
+                            let (accept_collateral, expiry_timestamp) = node
+                                .inner
+                                .get_collateral_and_expiry_from(dlc_channel.channel_id)?;
+
+                            position::handler::update_position_after_dlc_creation(
+                                order,
+                                accept_collateral,
+                                expiry_timestamp,
+                            )?;
+
+                            tracing::info!("Successfully recovered position from order.");
+                        }
+                        Err(e) => {
+                            tracing::error!("Could not recover position from order as no filling order was found! Error: {e:#}");
+                        }
+                    }
+                } else if matches!(
+                    dlc_channel.state,
+                    SubChannelState::Offered(_)
+                        | SubChannelState::Accepted(_)
+                        | SubChannelState::Finalized(_)
+                ) {
+                    recover_dlc().await?;
+                } else {
+                    tracing::warn!("The DLC is in a state that can not be recovered.");
+                }
+            }
+        }
+        _ => {}
+    }
+
+    Ok(())
+}
+
+/// Sends a pending RecoverDlc background task notification to the UI, allowing the UI to show a
+/// dialog with a spinner that the DLC protocol is still in progress.
+/// Also triggers the `periodic_check` to process any actions that might have been created after the
+/// channel reestablishment.
+///
+/// fixme(holzeis): We currently use different events for show the recovery of a dlc and the waiting
```

Comment:
🙏 again, please consider using standard uppercase `FIXME` instead of custom one (with your name added to it). see other comment.

Reply:
can you elaborate on the custom one? 😅 Is there a best practice that you are referring to 🙂. Happy to change it, but as describe above to me it only adds value seeing who added the `todo` / `fixme`